mino/proxy/http/controller/action.go

```diff
@@ -0,0 +1,28 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+
+	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/dela/mino/proxy/http"
+)
+
+type startAction struct{}
+
+// Execute implements node.ActionTemplate. It starts and injects the proxy http
+// server.
+func (a startAction) Execute(ctx node.Context) error {
+	addr := ctx.Flags.String("clientaddr")
+
+	proxyhttp := http.NewHTTP(addr)
+
+	ctx.Injector.Inject(proxyhttp)
+
+	go proxyhttp.Listen()
+	time.Sleep(time.Second)
```

Comment:
why do we need a `time.Sleep` here? 
It looks a bit wrong.
Are we waiting for Listen() to be operational? If so, isn't there a status we can check?

Reply:
With net/http there is no nice way that I know to do that. We could also poll the GetAddr() until we have an address. For now 1 sec is the reasonable amount of time we choose to check if anything panics.

Reply:
My concern is that `time.Sleep` is brittle. If there's a temporary freeze, or anything happens in the system, the next line will fail.

We could do something like this, though, no? 

```suggestion
        for proxyhttp.GetAddr() == nil {
	    time.Sleep(time.Second)
	}
```

Of course we'd then need an overall timeout to make sure that we don't go in an infinite loop if for whatever reason the Listen() fails